deisctl/backend/fleet/list_units.go

```diff
@@ -72,6 +72,28 @@ var (
 	}
 )
 
+// unitPrefixes are compared to unit names, so "deisctl list" doesn't show "deis-demo.service",
+// for example. Update this list when adding a new Deis unit file.
+var unitPrefixes = []string{
```

Comment:
Could we pull this into a global variable? It'd be nice to have one master list we could use here and in refreshUnits.